en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -48,79 +48,80 @@ Window update
-- The main domain creates the Texture2D by acquiring the global object sharedCanvas (Canvas of the open data domain), and then creates the SpriteFrame through the Texture2D, thereby assigning the SpriteFrame to the Sprite to be displayed to the main domain, if the open data field is operational Function (for example, operations such as sliding, dragging, etc.), then the primary domain needs to get the sharedCanvas in real time in the update to refresh the sprite.
+- Create an open data context project to obtain user data through the relevant API, and create a display of ui according to your own needs. The subdomain must use a full screen window, maintaining the same design resolution and adaptation mode as the primary context.
+- The main context creates the Texture2D by acquiring the global object sharedCanvas (Canvas of the open data context), and then creates the SpriteFrame through the Texture2D, thereby assigning the SpriteFrame to the Sprite to be displayed to the main context, if the open data field is operational Function (for example, operations such as sliding, dragging, etc.), then the primary context needs to get the sharedCanvas in real time in the update to refresh the sprite.
 
 ## Module selection
 
-Since the code and resources of the WeChat open data domain cannot be shared with the primary domain, it is very sensitive to the package. The developer needs to set the [project module culling option] for the open data domain project (../getting-started/basics/editor -panels/project-settings.md). It should be noted that starting with v2.0.0, developers can't check the WebGL Renderer in open data domain projects. The Canvas Renderer must be checked because the open data domain only supports Canvas rendering. At the same time, the rendering components supported under Canvas rendering are also limited (the UI components are not limited) and currently only support:
+Since the code and resources of the WeChat open data context cannot be shared with the primary context, it is very sensitive to the package. The developer needs to set the [project module culling option](../getting-started/basics/editor-panels/project-settings.md) for the open data context project. It should be noted that starting with v2.0.0, developers can't check the WebGL Renderer in open data context projects. The Canvas Renderer must be checked because the open data context only supports Canvas rendering. At the same time, the rendering components supported under Canvas rendering are also limited (the UI components are not limited) and currently only support:
 
-- Sprite
-- Label
-- Graphics
-- Mask
+  - Sprite
+  - Label
+  - Graphics
+  - Mask
 
-**Primary domain code example: **
+**Main context code example: **
```

Comment:
There is a space after the colon, which will cause the following effect:
![image](https://user-images.githubusercontent.com/19702899/49422033-21123d80-f7cd-11e8-9084-8ca1ad0d65fb.png)